lib/testing/gpuSimulateCudaBasebandBeamformer.hpp

```diff
@@ -0,0 +1,61 @@
+/**
+ * @file
+ * @brief CPU version of CUDA baseband beamforming kernel
+ */
+
+#ifndef SIMULATE_CUDA_BASEBAND_BEAMFORMER_HPP
+#define SIMULATE_CUDA_BASEBAND_BEAMFORMER_HPP
+
+#include "Config.hpp"          // for Config
+#include "Stage.hpp"           // for Stage
+#include "bufferContainer.hpp" // for bufferContainer
+
+#include <stdint.h> // for int32_t
+#include <string>   // for string
+
+/**
+ * @class gpuSimulateCudaBasebandBeamformer
+ * @brief Stage for faking CUDA baseband beamforming.
```

Comment:
Doc string should include the expected configuration and buffer parameters.

Reply:
Done!